src/handlers/pr_tracking.rs

```diff
@@ -0,0 +1,137 @@
+//! This module updates the PR work queue of the Rust project contributors
+//!
+//! Purpose:
+//!
+//! - Adds the PR to the work queue of one or more team member (when the PR has been assigned)
+//! - Removes the PR from the work queue of one or more team members (when the PR is unassigned or closed)
+//!
+//! Notes:
+//!
+//! - When assigning or unassigning a PR, we only receive the list of current assignee(s)
+//! - In case of a complete PR unassignment the list will be empty (i.e. does not have assignee(s) removed)
+//! - The code in this handler MUST be idempotent because GH emits a webhook trigger for every assignee
```

Comment:
> ... GH emits a webhook trigger for every assignee

This took me a bit to figure out. I'm not sure if the webhook could be configured differently.

Even we work with a single assignee, I think it's good to test everything to handling multiple assignees.